lib/ibm_watson/assistant_v1.rb

```diff
@@ -1897,10 +1897,11 @@ def list_dialog_nodes(workspace_id:, page_limit: nil, include_count: nil, sort:
     # @param digress_in [String] Whether this top-level dialog node can be digressed into.
     # @param digress_out [String] Whether this dialog node can be returned to after a digression.
     # @param digress_out_slots [String] Whether the user can digress to top-level nodes while filling out slots.
+    # @param disable [Bool] Whether to consider the dialog node during runtime evaluation. Set to `true` to ignore the dialog node.
     # @param user_label [String] A label that can be displayed externally to describe the purpose of the node to
     #   users. This string must be no longer than 512 characters.
     # @return [DetailedResponse] A `DetailedResponse` object representing the response.
-    def create_dialog_node(workspace_id:, dialog_node:, description: nil, conditions: nil, parent: nil, previous_sibling: nil, output: nil, context: nil, metadata: nil, next_step: nil, actions: nil, title: nil, node_type: nil, event_name: nil, variable: nil, digress_in: nil, digress_out: nil, digress_out_slots: nil, user_label: nil)
+    def create_dialog_node(workspace_id:, dialog_node:, description: nil, conditions: nil, parent: nil, previous_sibling: nil, output: nil, context: nil, metadata: nil, next_step: nil, actions: nil, title: nil, node_type: nil, event_name: nil, variable: nil, digress_in: nil, digress_out: nil, digress_out_slots: nil, disable: nil, user_label: nil)
```

Comment:
Mamoon, I just talked to @rrberry - `disabled` should only be in the dialog response, not in `create_dialog_node`. can you remove this?

Reply:
ok, we can just close the PR then.